docs/mocksupport.rst

```diff
@@ -600,6 +600,11 @@ There are two ways to build a mock repository:
   * Build from MOF definitions of the objects which are compiled into the
     repository. See :meth:`~pywbem_mock.FakedWBEMConnection.compile_mof_string`
     and See :meth:`~pywbem_mock.FakedWBEMConnection.compile_mof_file`.
+    If instances are to be compiled "instance of ..." they must be compiled
+    with the instance alias (ex:"instance of CIM_xxx as $xxx {") which adds
+    the CIMInstanceName with keybindings from the key properties to the
+    instance path. If compiled instances have duplicate paths, each new
```

Comment:
1. Suggest to change this sentence to:
"The MOF definition of any instances must specify an instance alias (ex: "instance of CIM_xxx as $xxx") which causes the instance path of that instance in the repository to be set up according to its key properties."

2. Why is the presence of the alias a precondition for building an instance path from the key properties? I suppose the alias name itself is not used in that process at all, so why can that not be done also for instances that do not have an alias? If you want to use the alias to safely identify the instances, it must be part of something in the repo. Where does the alias go in the repo?

Reply:
1. Made change.

2. Because I was uncomfortable building the instance path within the CreateInstance and there must be a path to act as the key in the repository.  It is the existance of the alias name that causes the path to be set in the instance and I did not want to mess with that.  No the only reason for setting the alias is that is what causes the path to be set into the instance. The alternative is to set the path in all createed instances and I was uncomfortable proposing that change as part of this particular issue. I created issue #1911 to marked discussion over issue of setting keybindings in new instances independent of the alias flag.

Reply:
OK, since issue #1911 was created, this comment can be considered done.